docs/adrs/0001-CRA-and-general-architecture.md

```diff
@@ -0,0 +1,50 @@
+<!-- Short descriptive title -->
+# UI Refresh General Architecture 
+
+<!-- Date -->
+*2021-10-19*
+
+<!-- Summary -->
+The new UI refresh needed basic decisions around infrastructure. Typescript/React was the framework desired. State management was also discussed; Redux was chosen due to it's focus on app-level state, and it's popularity. This ADR covers the architectural decisions made to support those frameworks.
+
+## Authors <!-- GitHub Username(s) -->
+@dauglyon
```

Comment:
I think it is good to make these links, so `[@dauglyon](https://github.com/dauglyon)` or similar. From what I can tell, it will not be automatically linked otherwise.

Reply:
```suggestion
[@dauglyon](https://github.com/dauglyon)
```